integration-tests/src/test/java/oracle/weblogic/kubernetes/ItOperatorWlsUpgrade.java

```diff
@@ -216,16 +216,6 @@ void testOperatorWlsUpgradeFrom341ToCurrent(String domainType) {
     installAndUpgradeOperator(domainType, "3.4.1", OLD_DOMAIN_VERSION, DEFAULT_EXTERNAL_SERVICE_NAME_SUFFIX);
   }
 
-  /**
-   * Auxiliary Image Domain upgrade from Operator v3.4.0 to current.
-   */
-  @Test
-  @DisplayName("Upgrade 3.4.0 Auxiliary Domain(v8 schema) Image to current")
-  void testOperatorWlsAuxDomainUpgradeFrom340ToCurrent() {
-    logger.info("Starting test to upgrade Domain with Auxiliary Image with v8 schema to current");
-    upgradeWlsAuxDomain("3.4.0");
-  }
-
   /**
    * Auxiliary Image Domain upgrade from Operator v3.4.1 to current.
    * The current release of the 3.4.1 does not conatin the resolution
```

Comment:
Please remove this comment as it's not required now.

Reply:
Removed the comments.